src/streamlink/plugins/tv999.py

```diff
@@ -0,0 +1,40 @@
+import re
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import validate
+from streamlink.stream import HLSStream
+
+
+class TV999(Plugin):
+    url_re = re.compile(r'https?://(?:www\.)?tv999\.bg/live\.html')
+    iframe_re = re.compile(r'<iframe.*src="([^"]+)"')
+    hls_re = re.compile(r'src="([^"]+)"\s+type="application/x-mpegURL"')
+
+    iframe_schema = validate.Schema(
+        validate.transform(iframe_re.search),
+        validate.get(1),
+        validate.any(None, validate.url()),
```

Comment:
If the regex search fails, it'll return a `None` value, and trying to access `[1]` on a `None` value will raise a `TypeError`. That's what the `validate.any(None, validate.all(...))` call is for.